attractions/autocomplete/autocomplete.svelte

```diff
@@ -39,6 +43,7 @@
     {maxOptions}
     {optionComponent}
     {...$$restProps}
+    on:change
```

Comment:
You keep forgetting about the rename before forwarding)

Reply:
If by _rename_ you mean wrapping the value into an object, then yes, you are right, fixed it

Reply:
I meant renaming the event to `nativeEvent`

Reply:
That one isn't native, it comes from the `AutocompleteField`

Reply:
oh, GitHub didn't show all the lines so I didn't notice 😅